Tests/TuistLoaderTests/Mappers/ProjectNameAndOrganizationMapperTests.swift

```diff
@@ -0,0 +1,51 @@
+import ProjectDescription
+import TuistCore
+import TuistSupport
+import XCTest
+
+@testable import TuistCoreTesting
+@testable import TuistLoader
+@testable import TuistSupportTesting
+
+class ProjectNameAndOrganizationMapperTests: TuistUnitTestCase {
+    var subject: ProjectNameAndOrganizationMapper!
+    var config: TuistCore.Config!
+
+    override func setUp() {
+        let nameTemplate: TemplateString = "Tuist-\(.projectName)"
+        config = TuistCore.Config.test(generationOptions: [
+            .xcodeProjectName(nameTemplate.description),
```

Comment:
minor:

Does this not work inlined?

```swift
.xcodeProjectName("Tuist-\(.projectName)")
```

Reply:
Nop. The `GenerationOption` in `TuistCore` expects the template to be a string.